docs/source/commandline/windowsconfig.rst

```diff
@@ -8,191 +8,69 @@ This section contains configuration options that are specific to the native (non
 Debugging options
 -----------------
 
-**-[no]oslog**
-
-	Outputs the error.log data to the Windows debugger. This can be used at	the same time as -log to output the log data to both targets as well. Default is OFF (*-nooslog*).
-
-**-watchdog** *<duration>* / **-wdog** *<duration>*
-
-	Enables an internal watchdog timer that will automatically kill the MAME process if more than *<duration>* seconds passes without a frame update. Keep in mind that some games sit for a while during load time without updating the screen, so *<duration>* should be long enough to cover that. 10-30 seconds on a modern system should be plenty in general. By default there is no watchdog.
+.. _mame-wcommandline-debuggerfont:
 
 **-debugger_font** *<fontname>* / **-dfont** *<fontname>*
 
 	Specifies the name of the font to use for debugger windows. By default,	the font is Lucida Console.
 
+.. _mame-wcommandline-debuggerfontsize:
+
```

Comment:
This option is also not Windows-specific, and the default depends on the debugger you choose.

Reply:
Also fixed this on my side.